packages/core-components/src/lib/unsorted/viz/Area.svelte

```diff
@@ -30,6 +30,10 @@
 
 	export let handleMissing = 'gap';
 
+	export let step = false; // shows step line. It can be true, false. Or 'start', 'middle', 'end' to configure turn point of step line.
+	$: step = step === 'true' || step === true;
```

Comment:
I'm not entirely sure I'm following, what is the expected behavior if step is `'start', 'middle', or 'end'` here?